api/common/modelstatus.go

```diff
@@ -25,6 +25,12 @@ func NewModelStatusAPI(facade base.FacadeCaller) *ModelStatusAPI {
 
 // ModelStatus returns a status summary for each model tag passed in.
 func (c *ModelStatusAPI) ModelStatus(tags ...names.ModelTag) ([]base.ModelStatus, error) {
+	// This call is used in both Controller and ModelManger facades.
+	// Luckily, prior to the signature change both of these facades were at version 3.
+	if bestVer := c.facade.BestAPIVersion(); bestVer < 4 {
```

Comment:
I'm not sure if this trace message is actually useful, as the client logic doesn't do anything different for version<4.